docs/Reference/Server.md

```diff
@@ -1105,6 +1106,15 @@ no new scope is created and therefore we have no place to attach contextual
 data. In that case, the plugin name will represent the boot order of all
 involved plugins in the format of `fastify -> plugin-A -> plugin-B`.
 
+#### hasPlugin
+<a id="hasPlugin"></a>
+
+Method to check if the plugin is registered.
```

Comment:
```suggestion
Method to check if a specific plugin has been registered.
Relies on the plugin metadata name.
Returns `true` if the plugin is registered. 
Otherwise, returns `false`. 
```

Reply:
@jsumners 
Thanks. I fixed it.
https://github.com/fastify/fastify/pull/3963/commits/8f33812e416116639d07576de2032424164931ad

Reply:
@jsumners 
Sorry, I added that.
https://github.com/fastify/fastify/pull/3963/commits/14e6ded427a1a58fbfba995a7a08b4e2f87b4c59